project_event/views/project_task_view.xml

```diff
@@ -1,15 +1,38 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <record id="project_event_task_tree_technical_sheet" model="ir.ui.view">
```

Comment:
You should pull the latest changes, this was removed and you are adding it back. Please Pull the latest 11.0_develop and use it as base

Reply:
@eilst Ok